test_util/cluster_api.py

```diff
@@ -326,7 +326,7 @@ def request_wrapper(self, request_fn, path, host_url=None, node=None, port=None,
         if node is not None:
             assert host_url is None, 'Cannot set both node and host_url'
             host_url = self.get_node_url(node, port=port)
-        return super().request_wrapper(request_fn, path, host_url=host_url, **kwargs)
```

Comment:
If I would have thought about this harder I would have realized this causes spawned clients to go this ClusterApi's super, not their super and thereby it has abi_base of ClusterApi's ApiClient and not whatever the client set for itself